remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
I pasted part of the error log above. in fact, without the correction (the 3rd one) the ^M can be seen in the log behind the "reading database". When run "ceph-deploy install" or "ceph-deploy purge", the error log can be seen easily.